### PR TITLE
add dictdefaultvalue.py and boolisslow.py

### PR DESCRIPTION
**Pull request by shiyanhui**
add setdefault and some fact about True and False in python2

### CODE REVIEW

**Comment by yann-shi** [comment]
@brennerm Could you please review and merge it? Thanks.

File: boolisslow.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+"""
+True and False are keywords in python3, but not in python2. In python2
+the value of True is 1, and the value of False is 0. True and False can be
+regarded as vars and thus they are slow in while loop.
+
```
Comment (brennerm) [inline boolisslow.py:5]: Please add an explicit hint that this leads to a performance improvement only in Python2.

**Comment by brennerm** [comment]
Sorry for letting you wait.
Please have a look at my comment.

File: boolisslow.py (hunk 1 above)
Comment (yann-shi) [inline boolisslow.py:5]: OK.

**Pull request merged**